clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/CommitEvent.java

```diff
@@ -29,8 +30,8 @@ public abstract class CommitEvent extends CompletableApplicationEvent<Void> {
      */
     private final Map<TopicPartition, OffsetAndMetadata> offsets;
 
-    protected CommitEvent(final Type type, final Map<TopicPartition, OffsetAndMetadata> offsets) {
-        super(type);
+    protected CommitEvent(final Type type, final Timer timer, final Map<TopicPartition, OffsetAndMetadata> offsets) {
```

Comment:
A nit perhaps but you've put the Timer as the final argument in all other cases, so I suggest you do it for this one too for consistency.

Reply:
Will do.

Reply:
Done